examples/fetch.ts

```diff
@@ -0,0 +1,80 @@
+/* tslint:disable:no-console */
+import Fluvio, { TopicReplicaParam, Offset } from '../src/index'
+import { v4 as uuidV4 } from 'uuid'
+
+// Set delay for creating a topic;
+async function sleep(ms: number) {
+    return new Promise((resolve) => {
+        setTimeout(resolve, ms)
+    })
+}
+
+// Set unique topic name
+const TOPIC_NAME = uuidV4()
+const PARTITION = 0
+
+// Define a topic spec;
+const TOPIC_SPEC = {
+    partitions: 1,
+    replicationFactor: 1,
+    ignoreRackAssignment: false,
+} as TopicReplicaParam
+
+// const MESSAGE_COUNT = 100
+
+async function consume() {
+    try {
+        const fluvio = new Fluvio()
+
+        // Explicitly call `.connect()` to connect to the cluster;
+        // This allows for lazily-loading the connection, useful in
+        // situations where the fluvio client does not need to immediately
+        // connect.
+        await fluvio.connect()
+
+        // Set the admin client to create and find topics;
+        const admin = await fluvio.admin()
+
+        // Create the topic
```

Comment:
leave topic creation to CLI or admin API in separate flow.  just assume topic exist or verify topic

Reply:
This is a copy/paste from a consumer example, hence follows existing workflow. 
Needs overhaul by engineering for consistency.